main.go

```diff
@@ -21,15 +20,23 @@ var (
 	versionString = fmt.Sprintf("%s, revision %s, branch %s, date %s; go %s", version.Version, version.Revision, version.Branch, version.BuildDate, version.GoVersion)
 
 	rootCmd = &cobra.Command{
-		Version: versionString,
-		Use:     "cy",
-		Short:   "Cycloid CLI",
-		Long:    `Cy is a CLI for Cycloid framework. Learn more at https://www.cycloid.io/.`,
+		Version:       versionString,
+		SilenceErrors: true,
+		SilenceUsage:  false,
+		Use:           "cy",
+		Short:         "Cycloid CLI",
+		Long:          `Cy is a CLI for Cycloid framework. Learn more at https://www.cycloid.io/.`,
 	}
 )
 
+func inRed(msg string) string {
+	return fmt.Sprintf("\033[1;31m%s\033[0m", msg)
+}
+
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
+		rootCmd.PrintErrln(inRed("Error:"), err.Error())
+		// rootCmd.PrintErrf("Run '%v --help' for usage.\n", rootCmd.CommandPath())
```

Comment:
comment

Reply:
Note discussed internally and we will merge this PR to provide a display of error details as it is blocking for our users.
We will then have a second iteration to redefine the error wrapping.

The goal would be to be able to detect if it's a Cobra (command syntax) error or an error from Cycloid API.
To display only the  `--help` when it's an issue from the command line